catkit2/services/thorlabs_mcls1_sim/thorlabs_mcls1_sim.py

```diff
@@ -117,7 +117,13 @@ def set_target_temperature(self, value):
         pass
 
     def get_power(self):
-        return self.testbed.simulator.light_source_data[self.id + '_power']
+        try:
```

Comment:
This is to circumvent the case where this key isn't set by the time this is called - need to figure out why that is.